perceval/backends/core/confluence.py

```diff
@@ -105,7 +107,7 @@ def search_fields(self, item):
 
         return search_fields
 
-    def fetch(self, category=CATEGORY_HISTORICAL_CONTENT, from_date=DEFAULT_DATETIME):
+    def fetch(self, category=CATEGORY_HISTORICAL_CONTENT, from_date=DEFAULT_DATETIME, max_contents=MAX_CONTENTS):
```

Comment:
```suggestion
    def fetch(self, category=CATEGORY_HISTORICAL_CONTENT,
              from_date=DEFAULT_DATETIME,
              max_contents=MAX_CONTENTS):
```

Reply:
You missed to explain the purpose of this parameter in the documentation of the `fetch` method.

Reply:
I see `spaces` is not available either, so please update it with it too.